### PR TITLE
list and host parameters are mutually exclusive

### PR DESCRIPTION
**Pull request by leki75**

### CODE REVIEW

File: openstack-installer/inventory/inventory.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import yaml
 import json
 
+
```
Comment (gyurco) [inline openstack-installer/inventory/inventory.py]: Remove this please.

File: openstack-installer/inventory/inventory.py [hunk 2]
```diff
@@ -56,10 +57,18 @@ def inventory(hostname):
...
-parser.add_argument('--host', help='returnt the variables for the hosts')
-args = parser.parse_args()
 
-inventory(args.host)
+def main():
+
+    parser = argparse.ArgumentParser(
+        description='Dynamic inventory for Openstack.')
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument('--list', help='list the hosts', action='store_true')
```
Comment (gyurco) [inline openstack-installer/inventory/inventory.py:65]: Unfortantely this gives up a nice feature: when run without any parameters, it just lists the inventory. (--list is the default)
Ansible doesn't require this, but it is convenient to see the inventory.

File: openstack-installer/inventory/inventory.py (hunk 1 above)
Comment (leki75) [inline openstack-installer/inventory/inventory.py]: Done

File: openstack-installer/inventory/inventory.py (hunk 2 above)
Comment (leki75) [inline openstack-installer/inventory/inventory.py:65]: Done

**Comment by gyurco** [comment]
Now it is 3 commits. One is more than enough :)

**Comment by leki75** [comment]
Done

**Comment by gyurco** [comment]
Ok, now let's leave these scripts alone for a while!

**Pull request merged**